docs/monitors/appmesh.md

```diff
@@ -131,80 +131,12 @@ The following table lists the metrics available for this monitor. Metrics that a
 | `upstream_rq_tx_reset` | cumulative |  | Total requests that were reset locally |
 
 
```

Comment:
Where is all the info about metricsToInclude going?

Reply:
It is going away and replaced by the added text below that links to the filtering doc that would have pointers to the legacy stuff but generally deemphasize it.